builtin/providers/openstack/resource_openstack_networking_secgroup_rule_v2.go

```diff
@@ -83,6 +88,16 @@ func resourceNetworkingSecGroupRuleV2Create(d *schema.ResourceData, meta interfa
 		return fmt.Errorf("Error creating OpenStack networking client: %s", err)
 	}
 
+	portRangeMin := d.Get("port_min_range").(string)
```

Comment:
It looks like there were a few issues with my rough sketch of this. This should be `d.Get("port_range_min").(int)`